<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>827</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Triangular</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks|QMainWindow::DockOption::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="side_v_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="storage_status_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(175, 227, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>재
료
현
황</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recipe_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(175, 227, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>레
시
피</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recommend_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(175, 227, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>추
천
페
이
지</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setting_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(175, 227, 255, 0);
border-color: rgba(255, 255, 255, 0);
</string>
        </property>
        <property name="text">
         <string>설
정</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTabWidget" name="tab_root">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="storage_tab">
         <attribute name="title">
          <string>재료</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>465</width>
               <height>842</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QScrollArea" name="scrollArea_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>447</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_4">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>584</width>
                   <height>158</height>
                  </rect>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="1" column="3">
                   <widget class="QTextBrowser" name="textBrowser_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QTextBrowser" name="textBrowser_7">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QTextBrowser" name="textBrowser_4">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QTextBrowser" name="textBrowser_6">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QTextBrowser" name="textBrowser_5">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QTextBrowser" name="textBrowser_3">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="4">
                   <widget class="QTextBrowser" name="textBrowser_8">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>137</width>
                      <height>67</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>447</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_6">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>445</width>
                   <height>158</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>447</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_7">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>445</width>
                   <height>158</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea_8">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>447</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_8">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>445</width>
                   <height>158</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>447</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_5">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>445</width>
                   <height>158</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="recipe_tab">
         <attribute name="title">
          <string>레시피</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>120</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>ㄴㅁㅇㄴㅁ</string>
          </property>
          <property name="placeholderText">
           <string>ㅁㅇㄴㅁㅇ</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="recommend_tab">
         <attribute name="title">
          <string>추천</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>110</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>ㅊㅊㅋㅊㅌ</string>
          </property>
          <property name="placeholderText">
           <string>ㅁㅇㄴㅁㅇ</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="setting_tab">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <attribute name="title">
          <string>설정</string>
         </attribute>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>100</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>유융뉴ㅠ</string>
          </property>
          <property name="placeholderText">
           <string>ㅁㅇㄴㅁㅇ</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
